velocity-transition-group.js

```diff
@@ -281,27 +283,39 @@ var VelocityTransitionGroup = React.createClass({
     // animations) so that we can then animate out. Velocity typically makes these transitions
     // very smooth, correctly animating from whatever state the element is currently in.
     if (entering) {
-      completeFn();
-      completeFn = null;
+      doneFn();
+      doneFn = null;
     } else {
       Velocity(nodes, 'stop');
     }
 
+    var combinedCompleteFn;
+    if (doneFn && opts.complete) {
+      var optsCompleteFn = opts.complete;
+      combinedCompleteFn = function () {
+        doneFn();
+        optsCompleteFn();
+      };
+    } else {
+      // One or the other or neither.
+      combinedCompleteFn = doneFn || opts.complete;
```

Comment:
`_finishAnimation` is used to jump to the end of the animation immediately, taking no time. We always finish the "leave" animation before triggering the "enter" animation in order to give "enter" a consistent starting point of "not there." But, when that's done, it doesn't make a lot of sense to trigger any "complete" callback that is associated with the element leaving, because it's actually about to appear!